index.html

```diff
@@ -199,15 +199,146 @@ <h1 class="header">What You Can Do at Technica</h1>
           </li>
         </ul>
       
-        <div class="tab-content">
+        <div class="tab-content unexpanded" onclick="expand(this)">
           <div id="friends" class="tab-pane fade in active show">
-            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+            <div class="content-wrapper row">
+              <div class="col-lg-6">
+                <div class="content-heading">Mini Events</div>
+                <div class="content-text">Want to take a break from coding and have some fun? Mini Events are for you!
+                  Mini-Events are a fun way to take a brain break during the weekend, and allow you to meet new friends, get
+                  creative,
+                  and even make something you can take home! A few Mini Events we already have planned for this year include
+                  Open Mic
+                  Night,
+                  Family Feud, DIY Your Own Technica Swag, and Board Game Corner.</div>
+              </div>
+              <img class="content-img col-10 col-md-5 img-fluid" src="./img/minievents_cream.svg" />
+            </div>
+            <div class="row expand-btn">Read More
```

Comment:
love this! if i click "read less" to collapse the section, the site jumps to the middle of the testimonials carousel, rather than staying on the What You Can Do section, which is a bit disorienting. this could be remediated by linking to gotechnica.org/#what-you-can-do

can you add these jump links for all sections as well, since we'll need to add them later anyway? you've prob made these before, but [here's a resource](https://blog.hubspot.com/marketing/jump-link-same-page) if not